src/modals/import-account-modal/import-account-seed/import-account-seed.tsx

```diff
@@ -61,11 +62,15 @@ export const ImportAccountSeed: FC<Props> = ({ onBackHandler }) => {
               <Label label="Seed phrase" description="Mnemonic. Your secret 12 - 24 words phrase." />
               <FormMnemonicInput name="seedPhrase" />
             </View>
-            <Divider size={formatSize(12)} />
-            <Disclaimer
-              title="Attention!"
-              texts={['Be aware that some third-party keyboards may capture your input data.']}
-            />
+            {isAndroid && (
+              <>
+                <Divider size={formatSize(12)} />
+                <Disclaimer
+                  title="Attention!"
+                  texts={['Be aware that some third-party keyboards may capture your input data.']}
+                />
+              </>
+            )}
```

Comment:
create the component and reuse it (use 16 divider everywhere)

Reply:
Agree